PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Related Tickets
+
+- [#TicketID](https://dev.sun-asterisk.com/issues/xxx)
+
+## What's this PR do ?
+
+- [x] Implement function A
+- [x] Fix bugs on function B
+
+## Library
+*(List package, library third party add new include version)*
+
+- Library: google-analysis.js
+
+## Performance
+
+- [ ] Resolved n + 1 query
+- [ ] Run explain query already
+- [ ] Time run rake task : 1000 ms
+
+## Checklist
+
+- [ ] It was tested in local success?
+- [ ] Fill link PR into ticket and the opposite
+
```

Comment:
a nghĩ nên thêm thằng này nữa
- [ ] Note reason, solution, scope of influence into ticket

Reply:
e thêm vào rồi nhé a